pkg/action/actioncontroller.go

```diff
@@ -121,6 +121,9 @@ func (ac *ActionController) handle(ctx context.Context, action *storkv1.Action)
 			ac.updateStatus(action, storkv1.ActionStatusSuccessful)
 		}
 	case storkv1.ActionTypeFailover:
+		if action.Status.Stage == storkv1.ActionStageFinal {
+			return nil
+		}
 		log.ActionLog(action).Infof("in stage %s", action.Status.Stage)
```

Comment:
Let's just put the action name too in this log.

Reply:
Already taken care. Explained in the below comment